PublicSDKAPITester/RCPurchasesErrorCodeAPI.m

```diff
@@ -0,0 +1,53 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  RCPurchasesErrorCodeAPI.m
+//
+//  Created by Madeline Beyl on 9/7/21.
+
+#import "RCPurchasesErrorCodeAPI.h"
+@import Purchases;
+
+@implementation RCPurchasesErrorCodeAPI
+
++ (void)checkEnums {
+    RCPurchasesErrorCode errCode = RCUnknownError;
+    switch(errCode) {
+        case RCUnknownError:
```

Comment:
I think it'd make sense to also check the values for these, we've had issues in the past where we messed up the values for errors and that translates into a bad mapping for the hybrids

Reply:
Makes sense.

Reply:
I think we'll need to add that to unit tests.

Reply:
Added unit tests for this in RevenueCat project (ErrorCodeTests.swift)